ios/packages/test-utils/Sources/ui-test/MockFlowManager.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  MockFlowManager.swift
+//  PlayerUI
+//
+//  Created by Zhao Xia Wu on 2023-10-26.
+//
+
+import Foundation
+
+public class MockFlowManager: FlowManager {
```

Comment:
This seems very similar to [ConstantFlowManager](https://github.com/player-ui/player/blob/main/ios/packages/swiftui/Sources/ManagedPlayer/ConstantFlowManager.swift) can they be converged?